traits/has_traits.pyi

```diff
@@ -152,8 +152,7 @@ class Vetoable(HasStrictTraits):
 
 VetoableEvent: _Any
 
-class MetaInterface(ABCMetaHasTraits):
-    def __call__(self, adaptee: _Any, default: _Any = ...): ...
+class MetaInterface(ABCMetaHasTraits): ...
```

Comment:
The `__call__` signature was causing complaints about disagreement with the parent class. The plan for 7.0 is to get rid of the (long deprecated) special behaviour of `MetaInterface.__call__` anyway, but I'll do that in a separate PR.

Reply:
> I'll do that in a separate PR.

See #1719